corehq/apps/hqcase/utils.py

```diff
@@ -21,6 +21,12 @@
 import six
 
 SYSTEM_FORM_XMLNS = 'http://commcarehq.org/case'
+EDIT_FORM_XMLNS = 'http://commcarehq.org/case/edit'
+
+SYSTEM_FORM_XMLNS_MAP = {
+    SYSTEM_FORM_XMLNS: 'CommCareHQ System Form',
+    EDIT_FORM_XMLNS: 'CommCareHQ Data Correction',
```

Comment:
Isn't it the user who made the data correction? I think just `Data Correction Form` might be better, otherwise it sounds like this was something that happened automatically.

Reply:
good point, updated